<!-- -*- xml -*- -->
<launch>
  <!-- % rosrun usb_cam usb_cam_node _video_device:="/dev/video0" _pixel_format:="yuyv" _io_method:="mmap" _image_height:=640 _image_height:=480 -->
  <node pkg="usb_cam" type="usb_cam_node" name="camera" output="screen" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
	<param name="pixel_format" value="yuyv" /> 
	<param name="io_method" value="mmap"/>
	<param name="camera_info_url" value="package://visp_ros/tutorial/grabber/ros/cam_parameters.yml" />
  </node>

  <!-- % rosrun image_view image_view image:=/my_usb_cam_grabber/image_raw -->
  <node pkg="image_view" type="image_view" name="my_image_raw_viewer" args="image:=/camera/image_raw">
 	<param name="image_transport" value="compressed" /> 
  </node> 
</launch>


